static/User.py

```diff
@@ -32,4 +33,7 @@ def setPin(self, pin):
         self.pin = self.encode(pin)
 
     def setTwitterToken(self, token):
-        self.twittertoken = token
\ No newline at end of file
+        self.twittertoken = token
+
+    def setTime(self,wantsTime):
+        self.time=wantsTime
```

Comment:
For this we would need changes to the current DB. Make a card on Trello and assign it to all the backend people